.circleci/config.yml

```
@@ -4,12 +4,14 @@ workflows:
   version: 2
   build_and_test_all:
     jobs:
-      - build-test-linux
+      - build-test-linux:
+          context:
+            - hello-world-demos
 
 jobs:
   build-test-linux:
     docker:
-      - image: circleci/python:2 # same image restwrappers use
+      - image: cimg/python:3.9.9
     steps:
       - checkout
       - setup_remote_docker
```

Looks like this hasn't become a problem for you yet, but Eli and I [recently discovered](https://launchdarkly.slack.com/archives/C8130U6RE/p1650051230726309) that `setup_remote_docker` uses an old version of the Docker daemon by default, which [may not be compatible](https://discuss.circleci.com/t/fail-to-build-images-with-docker-compose/43678) with very recent versions of CircleCI docker images. So bumping that version might save us some headache later:
```suggestion
      - setup_remote_docker:
          version: 20.10.14
```

---

Hmm, I get this: `failed to create host: rpc error: code = InvalidArgument desc = Image docker-20.10.14 is not supported`

---

Sorry, I'm not sure where I got that value from, but looks like this is actually the max version you can set: `20.10.12`